experiments/genesis/openmp/ramble.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ramble:
+  include:
+  - ./configs/spack.yaml
+  - ./configs/variables.yaml
+  - ./configs/modifier.yaml
+
+  config:
+    deprecated: true
+    spack_flags:
+      install: '--add --keep-stage'
+      concretize: '-U -f'
+
+  modifiers:
+  - name: allocation
+
+  applications:
+    genesis:
+      workloads:
+        DHFR:
+          env_vars:
+            set:
+              OMP_NUM_THREADS: '{omp_num_threads}'
+          variables:
+            n_ranks: '{processes_per_node} * {n_nodes}'
+            experiment_setup: ''
+            processes_per_node: ['{sys_cores_per_node} / 6']
+            n_nodes: ['1']
+            omp_num_threads: ['{processes_per_node} / 2']
+            arch: 'OpenMP'
+          experiments:
+            genesis_omp_{n_nodes}_{processes_per_node}_{omp_num_threads}:
+              variables: {}
+  spack:
+    concretized: true
```

Comment:
I think you just need to remove `concretized: true` for this to pass the tests: see https://github.com/LLNL/benchpark/pull/265#pullrequestreview-2105221682